Test/comp/compile1quiet/CompileRunQuietly.dfy

```diff
@@ -1,5 +1,5 @@
 // RUN: %dafny /compileTarget:cs "%s" > "%t"
-// RUN: dotnet CompileRunQuietly.dll >> "%t"
+// RUN: dotnet %S/CompileRunQuietly.dll >> "%t"
```

Comment:
Do these test changes verify that the bug has been fixed?

Reply:
I would say so. Before they were testing for the previous, wrong, behavior, and now fail without the changes to the compiler.

Reply:
I see 👍